FantasmoSDK/Classes/FMLocationManager.swift

```diff
@@ -72,9 +72,12 @@ open class FMLocationManager: NSObject, FMApiDelegate {
     private var lastFrame: ARFrame?
     private var lastLocation: CLLocation?
     
-    private weak var delegate: FMLocationDelegate?
+    private var delegate: FMLocationDelegate?
+    private var isConnectedToServer: Bool = false
+    
+    /// A  boolean value that states whether location updates were started by invoking `startUpdatingLocation()`.
+    public var isLocationUpdateInProgress = false
```

Comment:
This seems to denormalize the state information we are already storing in `.state`. I believe we do not need it, as this information can be calculated based on `.state`.

Reply:
Yes, totally agree. Have corrected.